.github/workflows/android.yml

```diff
@@ -0,0 +1,150 @@
+name: Android CI
+
+on:
+  push:
+    branches:
+      - master
+      - '!gh-pages'
+    paths:
+      # Rebuild when workflow configs change.
+      - .github/workflows/**
+      # Or when kotlin code changes.
+      - kotlin/**
+  pull_request:
+    paths:
+      - .github/workflows/**
+      - kotlin/**
+
+jobs:
+  assemble:
+    name: Assemble
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      # These setup steps should be common across all jobs in this workflow.
```

Comment:
Kind of aggravating that these need to be copy / pasted all over the place. Oh well.

Reply:
Tell me about it.